---
title: "Research"
layout: gridlay
sitemap: false
permalink: /research/
---

<style>
img{
  border-radius: 10px;
}
.col-md-3 {
  margin-top:10px;
  margin-bottom:10px;
  padding:0px;
  display:block;
  overflow:hidden;
  text-align:center;
  display: table-cell;
  background: white;
  border-radius: 20px;
  height: auto;
}
iframe {
  margin:0;
  padding:0;
  width: 175px;
  display: inline;
  vertical-align: middle;
}
</style>

## Research

<div class="jumbotron">
<div class="col-md-12 col-sm-12">
<h4><span style="color:#EFEFDA;"><b>Scope 1</b></span></h4>
<p style="text-align: justify;">
<span style="color:#1976d2;"><b>Deep learning-based discovery and engineering of novel gene-modification enzymes (transposases/integrases/methylases), and their applications in gene and cell therapy.</b></span><br/>
Deep learning is revolutionizing the discovery and engineering of novel gene-modification enzymes, such as Cas9, transposases, deaminases, and primer editors. This approach accelerates the process beyond traditional, trial-and-error methods, allowing researchers to explore a much broader biochemical space. The resulting enzymes are further tailored for advanced gene and cell therapies. For example, engineered integrases could enable more precise and durable integration of therapeutic genes, and optimized methylases could fine-tune gene expression for treating epigenetic disorders.
</p>
<div class="container">
<div class="row">
<center>
<img src="{{ site.url }}{{ site.baseurl }}/images/scope1.png" width="100%"/><br/>
<i>Copyright Reserved by Bang Wang's Lab (2025)</i>
</center>
</div>
</div>
<br/>

<h4><span style="color:#EFEFDA;"><b>Scope 2</b></span></h4>
<p style="text-align: justify;">
<span style="color:#1976d2;"><b>Application of protein language models (PLMs) in the evolution, in silico screening, and de novo generation of next-generation genome-editing tools.</b></span><br/>
Protein language models (PLMs) are transforming the development of next-generation genome-editing tools by leveraging vast datasets of protein sequences and structures to understand the rules of protein function. Instead of relying solely on natural evolution, PLMs enable in silico directed evolution, which guides the creation of improved enzymes for CRISPR-based systems and other editing technologies. Furthermore, PLMs facilitate the de novo generation of entirely novel genome editors. These models can design functional proteins that differ significantly from any known natural protein, offering new possibilities for therapeutic applications.
</p>

<h4><span style="color:#EFEFDA;"><b>Scope 3</b></span></h4>
<p style="text-align: justify;">
<span style="color:#1976d2;"><b>Enhancing high-throughput CRISPR target screening and engineering for immunotherapy cells (e.g., CAR-T/-NK cells).</b></span><br/>
High-throughput CRISPR screening is a powerful tool revolutionizing the engineering of therapeutic cell products, like CAR-T and CAR-NK cells. By systematically perturbing hundreds to thousands of genes at once, researchers can rapidly identify and validate key genetic targets that enhance the cells' anti-tumor efficacy, persistence, and safety. This technology is also being used to overcome challenges associated with the tumor microenvironment, identifying targets that can make these cells resistant to immunosuppressive factors. By leveraging CRISPR screening, scientists can accelerate the engineering of next-generation immunotherapy cells, facilitating "off-the-shelf" and personalized cancer treatments.
</p>
<div class="container">
<div class="row">
<center>
<img src="{{ site.url }}{{ site.baseurl }}/images/scope3.png" width="100%"/><br/>
<i>Copyright Reserved by Bang Wang's Lab (2025)</i>
</center>
</div>
</div>
<br/>

</div>
</div>
